hazelcast-jet-reference-manual/src/main/asciidoc/work-with-jet/log-debug.adoc

```diff
@@ -284,16 +288,27 @@ this:
 include::{javasource}/LogDebug.java[tag=s5]
 ----
 
-The handler for manipulating user metrics is the
+The handler for manipulating user-defined metrics is the
 `{jet-javadoc}/core/metrics/Metric.html[Metric]` class, instances can be
 obtained via the utility class
 `{jet-javadoc}/core/metrics/Metrics.html[Metrics]`). The `Metric`
 interface allows to both store simple measured values (like queue
 length, load-factor and such) and to incrementally count things (like
 number of certain events).
 
-User metrics can be used anywhere in pipeline definitions where custom
-code can be added. This means (just to name the most important ones):
+When obtaining `Metric` instances from the `Metrics` utility class it's
+possible to specify the desired `{jet-javadoc}/core/metrics/Unit.html[measurement unit]`
+of the metrics. It's important to note though that this measurement unit
+serves a purely informative purpose only (it's being used to set one of the
+tags of the metrics, for details see the
+`{jet-javadoc}/core/metrics/Unit.html[javadoc]`), no validation is being
+done on it. The `Metric` instance for a certain name can be requested multiple times
+from the utility class with different `Unit` values and this will not generate
+any errors. The first `Unit` value from the first request will take effect,
+subsequent ones will be ignored.
```

Comment:
The language is a bit garrulous:
- "It's important to note though that this measurement unit serves a purely informative purpose only" -> "The unit can be used by UI tools to format the value, it's ignored by Jet."
- "no validation is being done on it" -> already covered by "ignored by Jet".
- "The `Metric` instance for a certain ... will be ignored" -> "If you request a metric multiple times with different unit, the unit of the first request will be used to register the metric, others will be ignored"